src/databased/runtime.rs

```diff
@@ -268,14 +268,18 @@ impl Runtime {
             }
 
             Request::RemoveCheckpoint(swap_id) => {
-                self.database.delete_checkpoint_state(CheckpointKey {
+                if let Err(err) = self.database.delete_checkpoint_state(CheckpointKey {
                     swap_id,
                     service_id: ServiceId::Wallet,
-                })?;
-                self.database.delete_checkpoint_state(CheckpointKey {
+                }) {
+                    debug!("{} | Did not delete checkpoint entry: {}", swap_id, err);
```

Comment:
Maybe we could use `warn!` here?

Reply:
This happens too often, no need to warn. It's also just for deletion, so it is not that important.